### DISCUSSION TITLE
Where to report "broken" codesandbox examples from react-hook-form/examples?

### DISCUSSION THREAD

**Discussion by escapedcat** [react-hook-form/react-hook-form#7102] [General] [answered]
https://github.com/react-hook-form/react-hook-form/tree/master/examples#web  
Here the [Custom Input](https://codesandbox.io/s/react-hook-form-custom-input-hmyeu) throws 2 warnings on startup:

1. `Warning: The tag <lable> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.`
  Just a typo
2. `Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?`
  This is confusing for a react-hook-form beginner like me

Happy to help with this

**Comment by bluebill1049** [comment, marked as answer]
thanks that's fixed in the csb.

**Reply by escapedcat** [reply]
Wow, thanks a lot!
